profile_page/database/profile_page_db.py

```diff
@@ -280,63 +323,68 @@ def delete_user_game_graph(user_id : str, user_game_id: str):
     On Success:
         dictionary: original user game
     On Fail:
-        str: friendly response to infrom of an error
+        Boolean: False
     """
-    value_returned = client.InitialDB.User_Profile.find_one_and_update(
+
+    value_returned = client.InitialDB.User_Profile.find_one(
         {"user_id": user_id},
-        {"$set":{"save_games.$[index].graph":{}}},
-        upsert=False,
-        array_filters=[{"index.game_id":{"$eq": user_game_id}}]
+        {'_id':0,"save_games":{"$elemMatch":{"game_id":user_game_id}}}
     )
 
     if value_returned == None:
-        return 'nah bro idk about it'
-    return 1
+        return False
+    else:
+        try:
+            value_returned = value_returned['save_games'][0]
+            return value_returned
+        except:
+            return False
 
-def delete_user_game(user_id : str, user_game_id: str):
+def save_game( user_id : str, board : dict ):
```

Comment:
Let's add a new field in the user's profile information to list how many games the user saved so far.